torch_xla/distributed/spmd/api.py

```diff
@@ -0,0 +1,196 @@
+import logging
+import os
+from functools import wraps
+from typing import Any, Callable, Dict, Optional, Sequence, Tuple, Union
+
+import torch
+
+import torch.nn as nn
+from torch.distributed._tensor.device_mesh import DeviceMesh
+from torch.distributed._tensor.placement_types import Placement, Replicate
+
+log = logging.getLogger(__name__)
+
+TORCH_XLA_INITIALIZED = False
```

Comment:
I don't quite understand this `TORCH_XLA_INITIALIZED ` flag

Reply:
Ah, we don't need this anymore, due to the fact that this code now lives in torch_xla.